app/assets/stylesheets/components/_summary-card.scss

```diff
@@ -3,7 +3,7 @@
   width: 100%;
 
   &__body {
-    padding: 15px;
+    padding: 25px;
     width: 100%;
     @include box-shadow(0 1px 4px rgba(0,0,0,.3));
```

Comment:
Note that I used `#dfe5e8` for the border color because it's a variable already defined for other borders/lines (got it from the design). In the Sketch file, the border color is actually `#5f7d8c` at 0.2 opacity but I thought it was better not adding another variable for that because the two colors are pretty close (also keeping it consistent with other borders).